src/main/java/graphql/execution/ExecutionContext.java

```diff
@@ -59,6 +60,8 @@ public class ExecutionContext {
     private final ExecutionInput executionInput;
     private final Supplier<ExecutableNormalizedOperation> queryTree;
 
+    private final AtomicReference<DataLoaderDispatchStrategy> dataLoaderDispatcherStrategy = new AtomicReference<>(DataLoaderDispatchStrategy.NO_OP);
```

Comment:
could be just voilatile - its a one way set no?

Reply:
done